webclient/app/src/app/commons/multipleSelectChip/MultipleSelectChip.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react";
```

Comment:
MultipleSelectChip might be a misleading name. Maybe MultipleChipSelect would be a better one

Reply:
Das heißt im Beispiel auch genau so